bin/test/generate_test.dart

```diff
@@ -65,7 +65,7 @@ void main() {
               temporaryDirectory.path,
             ],
             workingDirectory: path.join(
-                DiagramGenerator.projectDir, 'utils', 'diagram_generator'),
+                DiagramGenerator.projectDir, 'packages', 'diagram_generator'),
```

Comment:
this test hasn't been run for a while it seems

Reply:
Oh, wow, yeah.  Looks like the bin/ tests weren't being run.

Here's a PR for that (and this fix): https://github.com/flutter/assets-for-api-docs/pull/234

Reply:
I already enabled the test in this pr.

Reply:
Oh, I didn't notice that part.  OK.  I'll just revert my PR.